src/main/java/com/deloitte/elrr/jpa/svc/PersonSvc.java

```diff
@@ -20,8 +20,7 @@
  *
  */
 
-@Service
-@Slf4j
+@Service @Slf4j
```

Comment:
_grumble_ _grumble_ robits _grumble_

Reply:
actually this wasn't the robots, it was just good old autoformat

Reply:
https://github.com/US-ELRR/elrrserviceentities/pull/30/commits/98593835392a60dc4fda263f5452a74497c8347f